template/tasks.py

```diff
@@ -215,6 +215,14 @@ def build_docs(ctx):
     ctx.run("poetry run mkdocs build --strict")
 
 
+@invoke.task
+def run_docs(ctx):
+    """
+    Run the Codon documentation locally
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Copy paste error, Codon was/is a specific project. 

```suggestion
    Run the project documentation locally
```

Reply:
Thanks for the heads up. I thought that's a slogan for dev's docs 😅